0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -148,46 +168,51 @@ Content-Type: application/json
 
 ### Payment Setup
 
-The sender uses the receiver details to create the ILP packet:
+The sender uses the receiver details to create the PSK2 payment:
 
-* The `account` in the ILP packet is the `destination_account` provided by the receiver
-* The `amount` is determined by the sender:
+* The `destination_account` should be used as the PSK2 destinationAccount.
+* The `shared_secret` should be used as the PSK2 sharedSecret.
+* If present, the `balance.maximum` SHOULD be used as the PSK2 chunked payment amount.
 
-    * The sender uses the `ledger_info.currency_scale` to determine the integer amount to include in the ILP packet (e.g. if they want the receiver to receive 10 units of `ledger_info.currency_code` on their ledger, a `ledger_info.currency_scale` value of 2 would indicate they should set the `amount` to 1000). In cases where the sender knows the integer amount the receiver should get, for example if that amount was requested by the receiver out of band, the sender does not need to use `ledger_info.currency_code` and `ledger_info.currency_scale`
-    * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or lower than the `minimum_destination_amount`, as this will be rejected by the receiver
+In a UI, the `ledger_info` and `receiver_info` objects (if present) can be used for display purposes. These objects can be manipulated by the receiver in any way, so amounts SHOULD be displayed in source units when possible.
 
-* The `data` is encoded using the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md):
+Note that the sender can send as many PSK2 payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
-    * The key is the `shared_secret` provided by the receiver
-    * Additional data may be included in the PSK details. SPSP data MUST be JSON.
-    * Private PSK headers SHOULD include `Content-Type: application/json` and `Content-Length: <byte length of data>`
+## Appendix A: Payment Pointer
 
-Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
+This is the recommended way to identify an SPSP receiver, and is intended to be the main form of identifier that users on Interledger will interact with. It can be used as a persistent identifier for a person or as a temporary identifier to represent an invoice, much like a bitcoin address.
```

Comment:
If this is the main identifier people are going to interact with, I think it definitely needs some version negotiation in the resolution protocol. We're not going to want to change the identifier so we should plan for different versions of SPSP and/or PSK.

Reply:
Right now, `$bob.example.com` resolves to `https://spsp.bob.example.com`, with the accept header `application/spsp+json`. The response returned has a content type of `application/spsp+json` as well.

If we want to add a breaking version to the protocol, we can either:

1. change the subdomain (SPSP goes to `spsp.bob.example.com` but new SPSP goes to `spsp-v4.bob.example.com`.

2. change the accept header (SPSP is returned when the accept header is `application/spsp+json` but SPSPv4 is returned for `application/spspv4+json`.

3. (potentially combined with 2) return a content type of `application/spspv4+json` in the `Content-Type` response, allowing the client to check and make sure that it received a supported SPSP version.

Reply:
> If this is the main identifier people are going to interact with, I think it definitely needs some version negotiation in the resolution protocol.

The resolution of an endpoint to query based on the identifier needs to be simple and stable. The interaction with the endpoint can then accommodate versioning (sent in the request headers, list of versioned endpoints in the initial response etc)